Neos.Flow/Documentation/TheDefinitiveGuide/PartV/ReleaseNotes/700.rst

```diff
@@ -0,0 +1,532 @@
+  | #  | Uri Pattern                                                               | HTTP Method(s) | Name                                                                                     |
+  +----+---------------------------------------------------------------------------+----------------+------------------------------------------------------------------------------------------+
+  | 1  | neos/media/browser/images(/{@action}).{@format}                           | any            | Neos.Media.Browser :: Image Browser                                                      |
+  | 2  | neos/media/browser/assets(/{@action}).{@format}                           | any            | Neos.Media.Browser :: Asset Browser                                                      |
+  | 3  | neos/media/browser/asset-proxies(/{@action}).{@format}                    | any            | Neos.Media.Browser :: Asset Proxy Service                                                |
+  | 4  | neos/media/browser/image-variant(/{@action}).{@format}                    | any            | Neos.Media.Browser :: ImageVariant Service                                               |
+  | 5  | media/thumbnail/{thumbnail}                                               | GET            | Neos.Media :: Thumbnail                                                                  |
+  | .. | ...                                                                       | ...            | ...                                                                                      |
+  | 78 | {node}robots.txt                                                          | any            | Neos.Seo :: robots.txt                                                                   |
+  +----+---------------------------------------------------------------------------+----------------+------------------------------------------------------------------------------------------+
+
+  Run ./flow routing:show <index> to show details for a route
+
+See `Access Route Parameters when resolving routes`_ for upgrade instructions.
+
+DBAL connection factory
+-----------------------
+
+tbd
+
+.. code-block:: php
+
+    class SomeClass {
+
+        /**
+         * @Flow\Inject
+         * @var \Doctrine\DBAL\Connection
+         */
+        protected $connection;
+
+
+StaticResource EEL Helper
+-------------------------
+
+.. code-block:: html
+
+    <!-- create static resource uri -->
+    <link rel="stylesheet" href={StaticResource.uri('Neos.Demo', 'Public/Styles/Main.css')} media="all" />
+
+    <!-- get static resource content -->
+    <style>{StaticResource.content('Neos.Demo', 'Public/Styles/Main.css')}</style>
+
+tbd
+
+Human readable labels and descriptions for Roles & Privileges
+-------------------------------------------------------------
+
+tbd
+
+https://github.com/neos/flow-development-collection/pull/2166
+
+Unidirectional OneToMany relations
+----------------------------------
+
+tbd
```

Comment:
```suggestion
Inside a single aggregate `OneToMany` relations are normally best modelled unidirectionally. Bidirectional relations always are harder to manage correctly and can easily lead to unintentional traversal of entity hierarchies with all the drawbacks.
Since Doctrines `OneToMany` annotation is always bidrectional and also dictates the owning side of the relation (at the unexpected side from a modelling PoV), it is not straightforward to model this correctly.

In Flow specifically, we try to follow DDD best practices in modelling and this means, that the aggregate root is the entry point and the entity that is sent to a repository to persist it and all its subentities. This can not be achieved with the standard doctrine `OneToMany` annotation when the one side is supposed to be closer to the root.

This change allows the user to annotate such a relation simply as:

.. code-block:: php

    /**
     * @ORM\OneToMany
     * @var Collection<Comment>
     */

This is done by remapping `OneToMany` annotations without a `mappedBy` as `ManyToMany` with a unique constraint.
```

Or to keep it more concise at this place:
```suggestion
In Flow we try to follow DDD best practices in modelling and this means, that the aggregate root is the entry point and the entity that is sent to a repository to persist it and all its subentities. This can not be achieved with the standard doctrine `OneToMany` annotation when the one side is supposed to be closer to the root.

This change allows the user to annotate such a relation simply as:

.. code-block:: php

    /**
     * @ORM\OneToMany
     * @var Collection<Comment>
     */
```